tasks/main.yml

```diff
@@ -1,15 +1,15 @@
 ---
 - name: Get tarfile
-  command: wget https://github.com/kr/beanstalkd/archive/v1.9.tar.gz -q -O /tmp/beanstalkd.tar.gz creates=/tmp/beanstalkd.tar.gz
+  command: wget https://github.com/kr/beanstalkd/archive/v{{ beanstalkd_version }}.tar.gz -q -O /tmp/beanstalkd.tar.gz creates=/tmp/beanstalkd.tar.gz
 
```

Comment:
Also add the version to the tmp-file, otherwise newer versions won't get downloaded